src/patchelf.cc

```diff
@@ -1748,6 +1753,8 @@ static void patchElf2(ElfFile && elfFile, const FileContents & fileContents, std
         elfFile.modifyRPath(elfFile.rpRemove, {}, "");
     else if (setRPath)
         elfFile.modifyRPath(elfFile.rpSet, {}, newRPath);
+    else if (addRPath)
+        elfFile.modifyRPath(elfFile.rpAdd, {}, newRPath);
```

Comment:
Should there be a conflict between add rpath and set rpath?

Reply:
Thanks for looking into this. I've just added a conflict when parsing arguments so you can only use one of them.